app/models/income.rb

```diff
@@ -5,4 +5,40 @@ class Income < ActiveRecord::Base
   def team_name
     self.team.team_name
   end
+
+  def credits
+    income_to_credit = get_income_to_credits_array
+    safe_get_credits_value(income_to_credit)
+  end
+
+  private
+  def get_income_to_credits_array
```

Comment:
Right now, the user/team structure is sort of in flux.  It's sort of a messed up design which needs to be rethought.  So it's fine to leave this here until.